client/jetpack-cloud/sections/partner-portal/license-product-card/style.scss

```diff
@@ -57,12 +97,58 @@
 	background: var( --studio-jetpack-green-50 );
 }
 
-/*
- * This is currently not being used, but should remain because it will be
- * added back in when we show prices for licenses.
- */
-p.license-product-card__text {
+@mixin license-product-card-block__pricing {
+	flex-basis: 100%;
+	margin: 1rem 0 0;
+	text-align: left;
+}
+
+@mixin license-product-card-line__pricing {
+	flex-basis: auto;
+	margin: 0 0 0 1rem;
+	text-align: right;
+}
+
+.license-product-card__pricing {
+	white-space: nowrap;
+	@include license-product-card-block__pricing;
+
+	@include break-mobile {
```

Comment:
(I don't think that I've ever suggested this before 😂 ): when we swap between two different layouts like this, should we perhaps add a "to and from" size query? So `@media (min-width: 123px) AND (max-width: 456px) {}`?

The upside is that if we add a new line of styling to e.g. `@mixin license-product-card-block__pricing`, then we wouldn't have to reset it in `@mixin license-product-card-line__pricing`

The downside is that it's not really following mobile first principles, and we would have to create a custom media query.
The custom media query is IMO partly OK since we can still use the breakpoint variables in `@import '@wordpress/base-styles/_breakpoints.scss';`.

---

I don't actually expect us to implement the above since it "feels" just as good to reset styling in the two different mixins. I just wanted to "throw it out there" ⚾

Reply:
I was torn between the two (and still am) but decided not to introduce new breakpoints right now.